akka-persistence/src/main/scala/akka/persistence/Persistence.scala

```diff
@@ -215,6 +253,34 @@ class Persistence(val system: ExtendedActorSystem) extends Extension {
     system.systemActorOf(pluginActorProps, pluginActorName)
   }
 
+  private def createAdapter(configPath: String): Option[ModelAdapter[Any]] = {
+    val pluginConfig = system.settings.config.getConfig(configPath)
+    if (!pluginConfig.hasPath("adapter")) None
+    else {
+      val adapterClassName = pluginConfig.getString("adapter")
```

Comment:
I might have completely misunderstood your "tagging prototype", but I thought you had something similar to the serialization extension that would support one tagger/adapter per configured event type? I thought that would be nice, but this is perhaps enough?